product/models.py

```diff
@@ -0,0 +1,98 @@
+from django.db import models
+
+class Category(models.Model):
+    name = models.CharField(max_length=50)
+
+    class Meta:
+        db_table = 'categories'
+
+class SubCategory(models.Model):
+    name     = models.CharField(max_length=50)
+    category = models.ForeignKey(Category, on_delete = models.SET_NULL, null = True)
```

Comment:
```suggestion
    category = models.ForeignKey(Category, on_delete = models.CASCADE)
```
Category가 없는 SubCategory는 존재할 수 없기 때문에, 삭제 옵션을 수정해야합니다. null=True 로 허용되면 안될 것 같네요. 확인해주세요